frontends/ast/genrtlil.cc

```diff
@@ -1126,8 +1126,14 @@ void AstNode::detectSignWidthWorker(int &width_hint, bool &sign_hint, bool *foun
 
 	// everything should have been handled above -> print error if not.
 	default:
-		for (auto f : log_files)
-			current_ast_mod->dumpAst(f, "verilog-ast> ");
+		// current_ast_mod is not set in e.g. packages
+		if (current_ast_mod) {
```

Comment:
This is a great idea. Thanks! In other places, we have used the following to grab the AST when processing a package.
```C++
AstNode *current_scope_ast = current_ast_mod == nullptr ? current_ast : current_ast_mod
```
This could simplify the code a bit, and hopefully provide additional useful information.

Reply:
Done.